### ISSUE TITLE
[REQUEST] Uranium Centrifuge for Duna series of parts

### ISSUE THREAD

**Issue by TDplai** [BobPalmer/MKS#1181]
As I mentioned on the forum thread:

> Anyways, I love this kolony/planetbase mod and I hope to see more (and the Karibou is awesome, despite not fitting in cargo bays) One feature that'd be great is a duna-series uraninite centrifuge!
-Me on KSP forum thread

I'd love to see 'Duna' series uranium centrifuges (and possibly reprocessing but for the range of parts for surface bases - is that really necessary?) because if you're using the Duna parts it kinda ugly-ifies your base (or mixed Duna/Karibou rover) to stick Tundra parts off the top (and in the case of rovers, really unbalances it)

**Comment by TDplai** [comment]
maybe have it a VERY big duna part (similar looks, but upscaled)?

**Comment by rspeed** [comment]
A Ranger series part might make more sense. So very compact, but with an extremely low output.

**Comment by TDplai** [comment]
True, duna parts do use ranger for processing, and it would go nicely on
the side of the duna power reactor, it would probably have to be
inflatable, inefficient and incapable of reprocessing depleted fuel for
balance and it could work as an efficiency part for when you get the
larger, more efficient and more capable tundra one.

**Comment by rspeed** [comment]
Wouldn't even need to be inflatable. Just something along the same lines as the other processing parts like the Regolith Sifter or Smelt-O-Matic. Maybe just one big disc shape sticking out of the center with some scienc-y bits attached to it.

**Comment by TDplai** [comment]
I guess inflatable or not is a matter suitable for a debate. On the one hand, Uraninite is a raw material, and ranger parts that handle raw material processing, but on the other hand, the only current way to make it is a Ranger 375 part, and that's huge, and most things that feed their output into a final usage are ranger inflatables.
I've thought a while, and came up with this:
A uraninite processor part would be not inflatable, like the smelt-o-matic but for uranium. 
Then if you want reprocessing going on, maybe you get an inflatable for that. A **large** inflatable.
But in the end, it's RoverDude's decision whether and/or how he implements this.
